README.md

```diff
@@ -30,12 +30,16 @@ My collected reading lists, reading recomendations, etc
 ### Management
 * Turn the Ship Around
 
+### Management
+* Turn the Ship Around
```

Comment:
you've added the Turn the Ship Around commit 17 days ago already, clean up your branches!

Reply:
:thinking: